tests/raftstore/server.rs

```diff
@@ -108,6 +108,12 @@ impl Simulator for ServerCluster {
 
         let mut cfg = cfg;
 
+        // Now we cache the store address, so here we should re-use last
+        // listening address for the same store.
+        if let Some(addr) = self.addrs.get(&node_id) {
```

Comment:
no need to get store address from pd every time, and in production, we can not change store address, so it is safe to cache it.